OracleFMWInfrastructure/README.md

```diff
@@ -128,7 +128,7 @@ The database is created with the default password `Oradoc_db1`. To change the da
 
```

Comment:
Add description about the new image Dockerfile-domain-in-image.
We build these images with script buildDockerImge.sh you need to change it so that it can build that image as well.